src/ekka_rlog_lb.erl

```diff
@@ -92,11 +92,12 @@ do_update() ->
 do_update(Shard) ->
     Timeout = application:get_env(ekka, rlog_lb_update_timeout, 300),
     CoreNodes = ekka_rlog:core_nodes(),
-    case rpc:multicall(CoreNodes, ?MODULE, core_node_weight, [Shard], Timeout) of
-        {[], _} ->
+    {Resp0, _} = rpc:multicall(CoreNodes, ?MODULE, core_node_weight, [Shard], Timeout),
```

Comment:
Is it ok to ignore badnodes here?

Reply:
Yes, we can't do anything meaningful about them here